app/src/main/java/org/oppia/app/help/HelpViewModel.kt

```diff
@@ -1,9 +1,6 @@
 package org.oppia.app.help
 
-import androidx.lifecycle.ViewModel
-import org.oppia.app.fragment.FragmentScope
-import javax.inject.Inject
+import org.oppia.app.viewmodel.ObservableViewModel
 
-/** [ViewModel] for [HelpFragment]. */
-@FragmentScope
-class HelpViewModel @Inject constructor() : ViewModel()
+/** [ObservableViewModel] for the recycler view of HelpActivity. */
+class HelpViewModel(public var title: String) : ObservableViewModel()
```

Comment:
Remove `public` word, by default it is public and also, instead of `var` use `val` as this value won't be changing.